pkgs/development/libraries/gettext/default.nix

```
@@ -63,7 +63,7 @@ stdenv.mkDerivation rec {
         mkdir -p "$oldFilesDir/$(dirname "$oldFile")"
         cp gettext-tools/gnulib-m4/extern-inline.m4 "$oldFilesDir/$oldFile"
       done
-      tar uf gettext-tools/misc/archive.dir.tar -C "$oldFilesDir" "''${oldFiles[@]}"
+      tar uf gettext-tools/misc/archive.dir.tar --sort=name --mtime="@$${SOURCE_DATE_EPOCH}" --owner=0 --group=0 --numeric-owner -C "$oldFilesDir" "''${oldFiles[@]}"
```

This comes out as `-mtime=@1{SOURCE_DATE_EPOCH}'` and tar complains about it:
```
tar: Substituting -9223372036854775807 for unknown date format '@1{SOURCE_DATE_EPOCH}'
```
```suggestion
      tar uf gettext-tools/misc/archive.dir.tar --sort=name --mtime="@$SOURCE_DATE_EPOCH" --owner=0 --group=0 --numeric-owner -C "$oldFilesDir" "''${oldFiles[@]}"
```

---

Oops, botched the escaping there indeed, thanks!